frontend/packages/kubevirt-plugin/src/topology/TopologyVmDetailsPanel.tsx

```diff
@@ -81,6 +83,14 @@ export const TopologyVmDetailsPanel: React.FC<TopologyVmDetailsPanelProps> = ({
       prop: 'templates',
       matchLabels: { [TEMPLATE_TYPE_LABEL]: TEMPLATE_TYPE_VM },
     },
+    {
```

Comment:
There is already a firehose for all VMs in TopologyDataController (registred by virtualMachineConfigurations in our plugin.ts).

It would be much nicer to reuse already existing loaded vms, and do not load the data twice.

Can we do something about this? Or is it out of the scope of this PR?